Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM --platform=${BUILDPLATFORM:-linux/amd64} node:14.17.0-alpine as builder
+FROM --platform=${BUILDPLATFORM:-linux/amd64} node:14.18.3-alpine as builder
```

Comment:
It might be better to just use `14-alpine` so that whenever the image is built using this `Dockerfile` the latest `14.x` version of Node is used.

Reply:
Agree, specially because 4.x is out of support, but I won't stop others to maintain it. I'd need to check if the release process still works on the 4.x branch, I think was disabled.